css/about.scss

```diff
@@ -131,7 +131,13 @@ h1, h2, h3, h4 {
     h4, i {
       color: #ff0000;
     }
-  }
+  }      
+  
+  .github {
+    h4, i {
+      color: #FFFFFF;
```

Comment:
I was going to comment about consistent casing for hex, but it seems like it's inconsistent elsewhere.
We should probably look into having some consistency and decide to use uppercase or lowercase.

Reply:
Even better, I'm planning a tidy up of SCSS and colours in future so this can be done in that

Reply:
So for now could be fine, then eventually resolved in #159 or if you have another issue / milestone for that?